src/app/store/configureStore.prod.js

```diff
@@ -7,13 +7,10 @@ import { routerMiddleware } from 'react-router-redux'
 
 export const history = createHistory()
 
-const enhancer = compose(
-  applyMiddleware(...[ thunk, routerMiddleware(history) ]),
-  storage()
-)
+const enhancer = compose(applyMiddleware(...[thunk, routerMiddleware(history)]), storage())
 
-export default function (initialState) {
-  if (initialState.router.location) {
+export default function(initialState) {
+  if (initialState.router.location && initialState.router.location) {
```

Comment:
this is wrong, it should look like dev file above ... However these changes should already be in your branch so maybe merge upstream development into this?